[2022-02-15 03:14:23,005] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.dbt_seed manual__2022-02-15T03:14:06.780838+00:00 [queued]>
[2022-02-15 03:14:23,021] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: dbt_training_bytecode.dbt_seed manual__2022-02-15T03:14:06.780838+00:00 [queued]>
[2022-02-15 03:14:23,022] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:14:23,023] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-15 03:14:23,024] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-15 03:14:23,045] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): dbt_seed> on 2022-02-15 03:14:06.780838+00:00
[2022-02-15 03:14:23,052] {standard_task_runner.py:52} INFO - Started process 1153 to run task
[2022-02-15 03:14:23,060] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_training_bytecode', 'dbt_seed', 'manual__2022-02-15T03:14:06.780838+00:00', '--job-id', '106', '--raw', '--subdir', 'DAGS_FOLDER/dbt-***.py', '--cfg-path', '/tmp/tmprb1uxiop', '--error-file', '/tmp/tmpwd0d0z50']
[2022-02-15 03:14:23,062] {standard_task_runner.py:77} INFO - Job 106: Subtask dbt_seed
[2022-02-15 03:14:23,163] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_training_bytecode.dbt_seed manual__2022-02-15T03:14:06.780838+00:00 [running]> on host 459d09dd7c86
[2022-02-15 03:14:23,272] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bytecode
AIRFLOW_CTX_DAG_ID=dbt_training_bytecode
AIRFLOW_CTX_TASK_ID=dbt_seed
AIRFLOW_CTX_EXECUTION_DATE=2022-02-15T03:14:06.780838+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-15T03:14:06.780838+00:00
[2022-02-15 03:14:23,275] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-15 03:14:23,278] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'dbt seed --profiles-dir /opt/***/dbt/dbt-training-bytecode-1 --project-dir /opt/***/dbt/dbt-training-bytecode-1']
[2022-02-15 03:14:23,295] {subprocess.py:85} INFO - Output:
[2022-02-15 03:14:30,364] {subprocess.py:89} INFO - 03:14:30  Running with dbt=1.0.1
[2022-02-15 03:14:30,950] {subprocess.py:89} INFO - 03:14:30  Found 8 models, 11 tests, 0 snapshots, 0 analyses, 367 macros, 0 operations, 1 seed file, 3 sources, 0 exposures, 0 metrics
[2022-02-15 03:14:30,958] {subprocess.py:89} INFO - 03:14:30
[2022-02-15 03:14:34,731] {subprocess.py:89} INFO - 03:14:34  Concurrency: 4 threads (target='dev')
[2022-02-15 03:14:34,736] {subprocess.py:89} INFO - 03:14:34
[2022-02-15 03:14:34,757] {subprocess.py:89} INFO - 03:14:34  1 of 1 START seed file dbt_bytecode_training.employees.......................... [RUN]
[2022-02-15 03:14:39,112] {subprocess.py:89} INFO - 03:14:39  1 of 1 OK loaded seed file dbt_bytecode_training.employees...................... [INSERT 4 in 4.35s]
[2022-02-15 03:14:39,199] {subprocess.py:89} INFO - 03:14:39
[2022-02-15 03:14:39,202] {subprocess.py:89} INFO - 03:14:39  Finished running 1 seed in 8.24s.
[2022-02-15 03:14:39,235] {subprocess.py:89} INFO - 03:14:39
[2022-02-15 03:14:39,238] {subprocess.py:89} INFO - 03:14:39  Completed successfully
[2022-02-15 03:14:39,240] {subprocess.py:89} INFO - 03:14:39
[2022-02-15 03:14:39,242] {subprocess.py:89} INFO - 03:14:39  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2022-02-15 03:14:39,646] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-15 03:14:39,674] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=dbt_training_bytecode, task_id=dbt_seed, execution_date=20220215T031406, start_date=20220215T031423, end_date=20220215T031439
[2022-02-15 03:14:39,716] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-15 03:14:39,768] {local_task_job.py:264} INFO - 3 downstream tasks scheduled from follow-on schedule check
